crypto-ffi/bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/core_cryptoFFI.modulemap

```diff
@@ -0,0 +1,4 @@
+module core_cryptoFFI {
+    header "core_cryptoFFI.h"
+    export *
+}
```

Comment:
Isn't this file auto-generated?

Reply:
No

Reply:
Right, but that's because `uniffi.toml` has `generate_module_map = false`.
If we remove that line, the modulemap file will be generated. So my question now is, why not just turn on modulemap generation?

Reply:
Actually my bad it was generated I'll delete it